api/app.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask
+from flask import request
+from flask import url_for
+from geodepy.geodesy import vincdir
+from geodepy.geodesy import vincinv
+from geodepy.convert import dms2dd
+from geodepy.convert import dd2dms
+
+app = Flask(__name__)
+
+coord2dd = {
+    'dd': lambda x: x,
+    'dms': dms2dd,
+}
+
+dd2coord = {
+    'dd': lambda x: x,
+    'dms': dd2dms,
+}
+
+
+@app.route('/')
+def list_routes():
+    return str(tuple(url_for(rule.endpoint) for rule in app.url_map.iter_rules() if rule.endpoint != 'static'))
+
+
+@app.route('/vincinv')
+def handle_vincinv():
+    from_coord = request.args.get('from_coord')
+    to_coord = request.args.get('to_coord')
+    lat1 = request.args.get('lat1', type=float)
+    lon1 = request.args.get('lon1', type=float)
+    lat2 = request.args.get('lat2', type=float)
+    lon2 = request.args.get('lon2', type=float)
```

Comment:
I don't really like the variable names above here. As a client invoking the API, it's not clear what lon1 is compared to lon2. Can these variables have more specific names or does that not make sense? Should from and to _coord be _coord_system or something? from_coord sounds like you're asking for a coordinate to me.

Reply:
I'm not sure on this one - maybe @BatchelorJ would have a better idea on this.

Reply:
I can see something like `from_coord_system` working though.

Reply:
lat1, lon1, lat2 and lon2 make sense as variable names in this case. It's also a hint to users that the inputs need to be in an angular notation form (dms, ddm, dec or hp). For the from and to _coord, I'd change that as it's a bit misleading. What's being specified by these variables is really the Angular Notation Type. What do you think of changing these to from and to _angle_notation or from and to _angle_type?